src/main/webapp/search-script.js

```diff
@@ -0,0 +1,84 @@
+function countSearchUnits() {
+  const inputElems = document.getElementsByName("search-units");
+  const arr = []
+  for (let i = 0; i < inputElems.length; i++) {       
+    if (inputElems[i].checked){
+      arr.push(inputElems[i].value);
+    }
+  }
+  return arr;
+}
+
+function countAddUnits() {
+  const inputElems = document.getElementsByName("num-units");
+  const arr = []
+  for (let i = 0; i < inputElems.length; i++) {       
```

Comment:
FYI in javascript you can use for of:
```
for(let elem of inputElems) {
 // Code
}
```

Usually cleaner.

Or in this case, this is a great case for doing an array filter then a map operation that would turn this into basically a one liner. That can wait for later though.

Reply:
oh that's awesome, didn't know you could do that, will implement right now (and later I'll do the map operation, definitely seems nicer)